rpc_jobs/standard_job_premerge_release.yml

```diff
@@ -0,0 +1,85 @@
+- project:
+    name: "RE-Release-PR"
+    repo:
+      - rpc-openstack:
+          URL: "https://github.com/rcbops/rpc-openstack"
```

Comment:
I don't know enough about the release process to know if this is possible, but would there be any value in using a fork of rcbops/rpc-openstack so that you could run through the full sequence of a release in a test scenario, without impacting rcbops/rpc-openstack in any way?

Reply:
The PR test does not make any changes, so there is no risk. It's a completely read-only set of tests.